cli_tools_tests/infra/no-external-ip/README

```diff
@@ -0,0 +1,9 @@
+Creates a GCP project that disables external IPs for VMs. Internet traffic is routed through
+a Cloud NAT router.
+
+Specifically the project has:
+ 1. No default network
+ 2. One network with default firewall
+ 3. One subnet (single region)
```

Comment:
Private Google access enabled or disabled?

Reply:
Private Google access is unmodified (disabled then?). AFAIK it isn't required since Cloud NAT is allowing access to the public endpoints.

Added text to clarify that PGA is not enabled.